src/main/java/jenkins/scm/api/SCMHead.java

```diff
@@ -155,47 +146,34 @@ public String toString() {
     }
 
     /**
-     * Gets all actions used to decorate the behavior of this branch.
-     * May be overridden to create a new list, perhaps with additions.
-     * @return a list of all actions associated with this branch (by default, an unmodifiable list searching {@link TransientActionFactory}s)
-     * @see Actionable#getAllActions
+     * Returns an empty list.
+     * @return an empty list
      * @since 1.1
+     * @deprecated this was added to the API in error. Retained for backwards binary compatibility only. Use
+     * {@link SCMSource#fetchActions(SCMHead, SCMHeadEvent, TaskListener)} to get the actions associated with a
+     * {@link SCMHead}
      */
+    @Restricted(DoNotUse.class)
+    @Deprecated
```

Comment:
We use `@Restricted` to flag methods for use via stapler bindings. I have the `@Deprecated` to differentiate this is something we really do not want you using. I'd much rather just remove the method